### PR TITLE
grouped commit from branch draw

### PR DESCRIPTION
**Pull request by kapapp**
Updated statistics, added proper commentary, added draw logic, changed ELO logic, made tests

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/blindwar/user/AppStatistics.kt [hunk 1]
```diff
@@ -11,89 +17,180 @@ class AppStatistics {
...
     }
 
+
     override fun toString(): String {
-        return "hello" +
-                //correctArray.toString() + wrongArray.toString() + correctPercent.toString() +
-                //wrongPercent.toString() +
-                wins.toString() + losses.toString() +
-                winPercent.toString() + lossPercent.toString() + elo.toString()
+        return "hello " + elo
```
Comment (CHOCOLVT) [inline app/src/main/java/ch/epfl/sdp/blindwar/user/AppStatistics.kt:194]: Hi!
why 'hello ' is added to elo? elo squared has a special meaning? haha

**Review by CHOCOLVT — APPROVED** [review]
good code factoring and kdoc
nice elo calculation function

File: app/src/main/java/ch/epfl/sdp/blindwar/user/AppStatistics.kt (hunk 1 above)
Comment (kapapp) [reply app/src/main/java/ch/epfl/sdp/blindwar/user/AppStatistics.kt:194]: Someone added a tostring that doesn't make much sense, especially with the arrays, so I changed that, but eventually we might just take it out

**Review by JorisMonnet — APPROVED** [review]
winPercent/lossPercent/drawPercent should be always calculated, not stored in appstatistics

great work !

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/MainMenuActivityTest.kt [hunk 2]
```diff
@@ -90,6 +89,9 @@ class MainMenuActivityTest : TestCase() {
         Thread.sleep(1000)
         onView(ViewMatchers.withId(R.id.profileButton))
             .perform(ViewActions.click())
+        onView(ViewMatchers.withId(R.id.emailView))
+            .perform(ViewActions.closeSoftKeyboard())
+        Thread.sleep(1000)
```
Comment (JorisMonnet) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/MainMenuActivityTest.kt:94]: I think that you don't need to sleep and you can just write closeSoftKeyboard out of a view

Comment (JorisMonnet) [reply app/src/androidTest/java/ch/epfl/sdp/blindwar/MainMenuActivityTest.kt:94]: I hope it works tho

Comment (kapapp) [reply app/src/androidTest/java/ch/epfl/sdp/blindwar/MainMenuActivityTest.kt:94]: Bruh I've slept like 2 nights this week

**Pull request merged**